### PR TITLE
Add R-scape recipe

### PR DESCRIPTION
**Pull request by JungeAlexander**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/rscape/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
+  run:
+    - libgcc    # [not osx]
+
+# fails for now due to a problem with R-scape dependency
+#test:
```
Comment (bgruening) [inline recipes/rscape/meta.yaml]: We can not merge it without a test. What was the error?

Comment (JungeAlexander) [inline recipes/rscape/meta.yaml]: There was a problem with parsing the `-h` argument in a script that is being called by R-scape. I added a test with a dummy Stockholm file. If the CI checks work, this should be a better test than just printing the help.

**Comment by bgruening** [comment]
Alex can you change the PR name? Is this ready? Looks good!

**Comment by JungeAlexander** [comment]
Yes, it's ready. I just changed the name and merged.

**Pull request merged**